### PR TITLE
update sidebar RSC-151

### PR DESCRIPTION
**Pull request by glenhunter**
https://issues.sonatype.org/browse/RSC-151

There were some discrepancies between what was described in the ticket and what was shown in the Zeplin file which I noted. I used the Zeplin file as the source of truth.

Current styling is in line with the Zeplin file now, however there is a new caret icon which is a change that should be attached to `NxTreeView` not the sidebar.

### CODE REVIEW

File: gallery/src/main.scss [hunk 1]
```diff
@@ -15,30 +15,31 @@
...
 .gallery-link {
+  @include regular();
+
   border-radius: 6px;
   box-sizing: border-box;
   display: inline-block;
-  height: 32px;
-  margin-bottom: 4px;
-  padding-left: 45px;
+  padding: $nx-spacing-xxs $nx-spacing-xs $nx-spacing-xxs 36px;
```
Comment (rpokorny) [inline gallery/src/main.scss:42]: What's the motivation for the non-standard number here?

Comment (glenhunter) [reply gallery/src/main.scss:42]: > What's the motivation for the non-standard number here?

With the sidebar spacing, and the indentation of the tree view this was the remainder to get to the indentation number specified in the designs

Comment (rpokorny) [reply gallery/src/main.scss:42]: > With the sidebar spacing, and the indentation of the tree view this was the remainder to get to the indentation number specified in the designs

Yes but why was a non-standard number specified in the designs?

Comment (glenhunter) [reply gallery/src/main.scss:42]: > > With the sidebar spacing, and the indentation of the tree view this was the remainder to get to the indentation number specified in the designs
> 
> Yes but why was a non-standard number specified in the designs?

From @dturner32's comment in the ticket, "In other words, it should be 52px from the left of the side bar container" 52 is not divisible by 8. As to why maybe @dturner can chime in?

**Pull request merged**